common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -205,7 +312,21 @@ div.problem {
       }
     }
 
-    &.incorrect, &.incomplete, &.ui-icon-close {
+    &.ui-icon-close {
+      p.status {
+        display: inline-block;
+        width: 20px;
+        height: 20px;
+        background: url('../images/incorrect-icon.png') center center no-repeat;
+      }
+
+      input {
+        border-color: red;
```

Comment:
Please use the `$red` variable or similar from `lms/static/sass/base_variables.scss`.